server/src/modules/events/domain/entities/__tests__/event.test.unit.ts

```diff
@@ -13,8 +13,19 @@ describe('Event AggregateRoot', () => {
     eventResult = Event.create({
       name: 'Event Name',
       description: 'Event Description',
+      url: 'Event URL',
+      startTime: new Date('2021-01-01'),
+      endTime: new Date('2021-01-01'),
+      facebookLink: 'Facebook',
+      twitterLink: 'Twitter',
+      instagramLink: 'Instagram',
+      websiteLink: 'Website',
+      backgroundImageURL: 'Background URL',
+      tags: ['tag1', 'tag2', 'tag3'],
+      clubs: [{ name: 'Club Name 1', iconURL: '' }],
```

Comment:
This seems like code we might repeat in the future when generating events; that raises the question, for larger objects like an Event, should we formalize the creation of factory (mock) methods so we can avoid convoluting tests with large creation calls? Not sure if this is actually necessary, since we have create-event-dtos already. This just might be a consequence of the unit test.